### DISCUSSION TITLE
Toast simple customization

### DISCUSSION THREAD

**Discussion by akiarostami** [chakra-ui/chakra-ui#2017] [General] [answered]
Hello,

I have a simple question regarding Toast. How can I add a margin to the top or right of a Toast, without rewriting the whole render function? Is that even possible?

Ideally, I want to do something like this: 

`toast({description: "Hello World" m={3}})`

Thank you.

**Comment by chasinhues** [comment, marked as answer]
@akiarostami sorry for the lack of response on our part! 

Looks like it will need to be done in the render method of the toast. You could create a reusable component that applies the styles you want in the render method, something like this - https://codesandbox.io/s/custom-toast-render-method-um7xg?file=/src/useToast.tsx

**Reply by akiarostami** [reply]
Thanks @ChasinHues!
